tests/unit_tests/model_annotation_tests/test_annotation.py

```diff
@@ -0,0 +1,20 @@
+from boimmgpy.model_annotation.model_annotator import LipidNameAnnotator
+from cobra.io import read_sbml_model,write_sbml_model
+
+
+def test_annotation(path):
+
+    model = read_sbml_model(path)
+    annotator = LipidNameAnnotator()
+    info = annotator.find_model_lipids(model)
+    print(info[3])
```

Comment:
you should put this inside a test case and this should be a method of the class. Check unittest package usage. You can see examples in the NEBiUM's workshop for testing